weaver/sdks/besu/node/package.json

```diff
@@ -39,5 +39,8 @@
   "publishConfig": {
     "registry": "https://npm.pkg.github.com/hyperledger"
   },
-  "tag": "latest"
+  "tag": "latest",
+  "overrides": {
+    "got": "12.4.1"
```

Comment:
Any reason to add this `got` dependency?

Reply:
HI @sandeepnRES, it's because the code would not compile otherwise.

Reply:
@krisstern Sounds like a valid reason but still a little short on the details though. Is it the ESM cut-over that's causing it where later versions of `got` are ESM-only?
If yes, it warrants some more investigation to see if the original declaration could be changed instead for example.
The larger point  being: Just getting it to compile isn't always enough. We have to make sure that the solution does not make the code less maintainable through technical debt (which using the overrides definitely **is**)

Reply:
Hi @petermetz Would that be acceptable if I update some of the versions for the packages in the `package.json`?

Reply:
@krisstern yes it perfectly fine to update the versions for packages as long as it builds and works as intended (for which CI tests are there, and one of them is failing right now)

Reply:
But on another point @krisstern I don't know what `got` package does but the code on main branch is stable and builds fine without the `got` package, 

So are you saying that it is required for this solution of removing `package-local.json`?

Reply:
Hi @sandeepnRES It could also be due to my setup. May I ask which `Node` version you are using for the project?

Reply:
CI tests are using node 18

Reply:
Yeah, I am using `Node 18` for the project too. So I am not sure why it didn't work for me before. Let me try again and see if I can do away with the override.

Reply:
Yes, thanks @petermetz!